### DISCUSSION TITLE
Where to find the arguments list?

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [neonbjb/tortoise-tts#310] [Q&A]
do_tts.py -h doesnt bring options

i dont see such documentation either

**Comment by kylereinfeld** [comment]
One option is to open the do_tts.py script in any editor and look at the list near the top of the file. All the arguments begin with parser.add_argument.

**Comment by Mazzaferno** [comment]
its in the api.py
            'ultra_fast': Produces speech at a speed which belies the name of this repo. (Not really, but it's definitely fastest).
            'fast': Decent quality speech at a decent inference rate. A good choice for mass inference.
            'standard': Very good quality. This is generally about as good as you are going to get.
            'high_quality': Use if you want the absolute best. This is not really worth the compute, though.
